src/components/File.jsx

```diff
@@ -93,39 +110,30 @@ class File extends Component {
           <button className='coz-btn coz-btn--extra' onClick={(e) => {
             onShowActionMenu(attributes.id)
             e.stopPropagation()
-          }} />
+          }} onpointerup={e => e.stopPropagation()} onpointerdown={e => e.stopPropagation()} />
         </div>
       </div>
     )
   }
 
-  renderFilenameInput (attributes) {
-    const classes = classNames(
-      styles['fil-content-cell'],
-      styles['fil-content-file'],
-      getClassFromMime(attributes)
-    )
-
-    return (
-      <div className={classes}>
-        <RenameInput />
-      </div>
-    )
-  }
-
-  renderFilenameCell (attributes, opening, canOpen) {
-    const { filename, extension } = splitFilename(attributes.name)
+  renderFilenameCell (attributes, opening, isRenaming) {
```

Comment:
I really don't know why this function is not a functional component. But let's say it is *historic*.